### PR TITLE
Add compatibility for RedHat family

### PR DESCRIPTION
**Pull request by nicomfer**

### CODE REVIEW

**Review by german-robles — APPROVED** [review]
LGTM

File: telegraf/map.jinja [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {% set oscodename = salt['grains.filter_by'](oscodename_map, grain='oscodename') %}
 
-{% if not grains['os'] == 'FreeBSD' %}
+{% if not grains['os'] == 'FreeBSD' and not grains['os_family'] == 'RedHat' %}
```
Comment (fzipi) [inline telegraf/map.jinja]: Why don't we use only for ubuntu, instead of listing all the ones we _don't want_?

File: telegraf/init.sls [hunk 2]
```diff
@@ -9,6 +9,15 @@ influxdata_repo:
     - file: /etc/apt/sources.list.d/influxdata.list
     - require_in:
       - pkg: telegraf
+{% elif grains['os_family'] == 'RedHat' %}
+influxdata_repo:
+  pkgrepo.managed:
+    - humanname: CentOS-$releasever - Base
```
Comment (fzipi) [inline telegraf/init.sls]: Maybe use the os name grain, instead of fixing it to `CentOS` ?

**Review by fzipi — CHANGES_REQUESTED** [review]
Added a couple comments.

File: telegraf/init.sls (hunk 2 above)
Comment (nicomfer) [reply telegraf/init.sls]: I've changed it to use grains OS

File: telegraf/map.jinja (hunk 1 above)
Comment (nicomfer) [reply telegraf/map.jinja]: Fixed to use just Debian OS Family

**Review by fzipi — APPROVED** [review]
LGTM.

**Pull request merged**